webapp/src/components/data_selectors/backend_selector.tsx

```diff
@@ -6,7 +6,7 @@ import React from 'react';
 import debounce from 'debounce-promise';
 import AsyncSelect from 'react-select/async';
 
-import {Theme} from 'mattermost-redux/types/preferences';
+import {Theme} from 'mattermost-redux/selectors/entities/preferences';
```

Comment:
Wouldn't this come from `@mattermost/types` and not `selectors`? Same with other instances of this

Reply:
This I have copied from the mattermost-plugin-calls. 
https://github.com/mattermost/mattermost-plugin-calls/blob/d4a96f4ceef5c12e55822b1d18195967dd071c59/standalone/src/theme_utils.ts#L2-L3

Reply:
Interesting, the `Theme` is indeed defined in [mattermost-redux/selectors/entities/preferences.ts](https://github.com/mattermost/mattermost/blob/4fbc96ec02cc07b2bc167e18afeeabaf30718f52/webapp/channels/src/packages/mattermost-redux/src/selectors/entities/preferences.ts#L115) and not [@mattermost/types/preferences.ts](https://github.com/mattermost/mattermost/blob/4fbc96ec02cc07b2bc167e18afeeabaf30718f52/webapp/platform/types/src/preferences.ts). @hmhealey Any ideas if this is intentional, and if it should be moved to `@mattermost/types`? And maybe or maybe not exported from `selectors` as well to keep backwards compatibility. Or we can view `mattermost-redux` as not following semver and make other projects reference `@mattermost/types`. Or we can just leave in `selectors`.

Reply:
Moving it to `@mattermost/types` makes sense to me. When we moved it to mattermost-redux a couple years ago, that had been our end goal. There's a couple weird/outdated fields in it that I wanted to get rid of, but I'd rather accept those if it helps get away from having plugins import from the web app directly